smart_contracts/contracts_as/test/do-nothing-stored/assembly/index.ts

```diff
@@ -32,9 +32,9 @@ export function call(): void {
     ACCESS_KEY_NAME,
   );
   const key = Key.create(CLValue.fromI32(result.contractVersion));
```

Comment:
Can we rename this variable, @Fraser999 has convinced me its best to avoid having the variable name be the same as the type itself